tests/TestCase.php

```diff
@@ -23,15 +23,15 @@ abstract class TestCase extends BaseTestCase
     /**
      * Call protected/private method of a class.
      *
-     * @param  string  $class Instantiated object that we will run method on
+     * @param  class-string  $class Instantiated object that we will run method on
```

Comment:
Not sure about this one. `class-string` is not a type..

Reply:
I think this a new phpstan feature.

Reply:
Yep, it's a phpstan feature, to indicate that it receives a class. 

Unfortunately phpstorm doesn't like it. 

Not sure if I am a fan of it though.